src/crimson/osd/osd_operation_sequencer.h

```diff
@@ -99,18 +166,18 @@ class OpSequencer {
   //          /--- unblocked (in pg pipeline)
   //         |      /--- blocked
   //         V      V
-  // |////|.....|.......| <--- last_issued
+  // |////|.....|.......| <--- last_seen
   //      ^     ^       ^
   //      |     |       \- prev_op
   //      |      \--- last_unblocked
   //      last_completed
   //
   // the id of last op which is issued
-  uint64_t last_issued = 0;
+  std::uint64_t last_seen_id = 0;
```

Comment:
hmm, i still think `last_issued` is a better name, but don't feel strong about this...

Reply:
Ah, now I see: _last issued **by the client of OpSequncer**_. At first glance I was struggling to understand that. I think my view was more `OpSequencer`-centric (_last seen **by the OpSequncer**_).

Still, no strong feelings on the `last_seen` neither.

Reply:
`last_seen` can be interpreted in multiple ways. if we consider this from the perspective of `OpSequencer`, what does "see" imply? if we can have a more accurate name, i'd be all for it.

Reply:
Hmm, basically it's the most recent op that `start_op()` has been called for. `last_started` maybe?

Nevertheless, I would be fine with reverting to `last_issued` too.

Reply:
`last_stared` is much better.

Reply:
Switching!